ariadne/src/Ariadne/Wallet/Backend/KeyStorage.hs

```diff
@@ -289,7 +289,7 @@ newAddress acidDb WalletFace {..} walletSelRef runCardanoMode accRef chain pp =
       , _hdAddressCheckpoints = one emptyAddrCheckpoint
       }
   throwLeftIO $ update acidDb (CreateHdAddress hdAddress)
-  walletRefreshState
+  addr <$ walletRefreshState
```

Comment:
Minor: personally, I believe using `<$` in non-`Functor` context is redundant because it requires remembering an extra operator whose equivalent is trivial:

```
walletRefreshState
pure addr
```

and arguably more readable.